static_src/scss/settings/_page.scss

```diff
@@ -83,22 +90,26 @@ h5 {
 
 @include mq($from: mobile) {
   h1 {
-    font-size: 1.8em;
+    font-size: 1.6em;
+  }
+
+  h2 {
+    font-size: 1em;
   }
 
   h3 {
-    font-size: 1.2em;
+    font-size: 0.8em;
   }
 }
 
 
 @include mq($from: desktop) {
   h1 {
-    font-size: 4em;
+    font-size: 1.6em;
```

Comment:
we might not need to adjust h tags for responsiveness - looks like the text is quite small from the design. TODO: either remove it or wait until we completely finish and see if need changes.